src/WarcraftLegacies.Source/Quests/Frostwolf/QuestHighmountain.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MacroTools.QuestSystem;
+using static War3Api.Common;
+using MacroTools.ObjectiveSystem.Objectives.LegendBased;
+using MacroTools.LegendSystem;
+
+namespace WarcraftLegacies.Source.Quests.Frostwolf
+{
+  /// <summary>
+  /// Bring any hero to the Echo Isles to unlock the base.
+  /// </summary>
+  public sealed class QuestHighmountain : QuestData
+  {
+    private readonly LegendaryHero _cairne;
+
+    /// <inheritdoc />
+    public QuestHighmountain(LegendaryHero cairne) : base("Invitation to a Feast",
```

Comment:
The quest name I'm suggesting is the one you get in WoW when you meet the High Mountain.
```suggestion
    public QuestHighmountain(LegendaryHero cairne) : base("A Feast for Our Kin",
```